### PR TITLE
Chapter 9 feedback fixes

### LINKED ISSUE #1100: Chapter 9 feedback
Some feedback for chapter 9

- [ ] Padding on the copy pages need to work better for responsive sizing
- [ ] Allow running the execute button to move the users focus to the execute tab in mobile
- [ ] Update the copy on the advanced challenges to flow better

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
Fixes #1100 

- padding fix
- switch to execute tab on mobile when running script
- came up with alternative advance challenge heading

### CODE REVIEW

**Comment by Emmanuel-Develops** [comment]
Regarding the padding issue, a cleaner approach would involve changing about **24 files.**
The changes in these files would mostly be removing the `my-8` class

example, from:
```jsx
<ChapterIntro
      className="my-8"
      heading={t('chapter_nine.opcodes_one.heading')}
    >
```

to:
```jsx
<ChapterIntro
      heading={t('chapter_nine.opcodes_one.heading')}
    >
```
This would simplify the styling, from styling based on some breakpoints to consistent padding across all screens.
With this disclaimer, reviewing it should be less of a chore, and I can proceed with that.
Is this fine?

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
       paragraph_two: `For each contract proposal you discuss, provide a bitcoin script and one valid spending stack.`,
     },
     proposal_two: {
-      title: 'Advanced Challenge 1',
+      title: 'Advanced Challenge: 2 of 2 multisig',
```
Comment (benalleng) [inline i18n/locales/en.ts:2581]:
```suggestion
      title: '2 of 2 multisig',
    ```

File: i18n/locales/en.ts [hunk 2]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
...
       nav_title: 'Sign cooperatively',
-      heading: 'Advanced Challenge 1',
+      heading: 'Advanced Challenge: 2 of 2 multisig',
```
Comment (benalleng) [inline i18n/locales/en.ts:2583]:
``` suggestion
      heading: '2 of 2 multisig',
```

File: i18n/locales/en.ts [hunk 3]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
...
       paragraph_one: `The first thing Vanderpoole suggests is a 2-of-2 multisig. All funds that get donated will be split by you and him 50/50, which will be managed by you both signing all spending transactions from the donation address, so you will have to agree on all "withdraws"`,
       paragraph_two: `Vanderpoole hands you his public key, it's PUBKEY(vanderpoole) and yours is PUBKEY(me).`,
       paragraph_three: 'Provide the initial stack to spend from the script.',
       next_step_message: 'Looks good! Now lets try with your own signature.',
     },
     proposal_three: {
-      title: 'Advanced Challenge 2',
+      title: 'Advanced Challenge: Condtional Time locked Transaction',
```
Comment (benalleng) [inline i18n/locales/en.ts:2590]:
```suggestion
      title: 'Conditional time locked transaction',
```

File: i18n/locales/en.ts [hunk 4]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
...
       nav_title: 'Make him wait',
-      heading: 'Advanced Challenge 2',
+      heading: 'Advanced Challenge: Condtional Time locked Transaction',
```
Comment (benalleng) [inline i18n/locales/en.ts:2592]:
```suggestion
      heading: 'Conditional time locked transaction',
```

File: i18n/locales/en.ts [hunk 5]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
...
       paragraph_one: `Wait a minute, that doesn't make sense -- you don't want to deal with him forever! The new deal is, you get all donations for the next two hours while you are still on TV. After that, he can have whatever else trickles in. You take a look at The Bitcoin Block Clock on the wall in the studio and agree that block height 6930300 will be probably be mined in about two hours`,
       paragraph_two: `Remember Vanderpoole's public key, it's PUBKEY(vanderpoole) and yours is PUBKEY(me).`,
       paragraph_three: 'Provide the initial stack to spend from the script.',
       next_step_message: 'Looks good! Now lets try with your own signature.',
     },
     proposal_four: {
-      title: 'Advanced Challenge 3',
+      title: 'Advanced Challenge: Conditional Hash Locked Transaction',
```
Comment (benalleng) [inline i18n/locales/en.ts:2599]:
```suggestion
      title: 'Secret preimage locked transaction',
```

File: i18n/locales/en.ts [hunk 6]
```diff
@@ -2578,27 +2578,27 @@ const translations = {
...
       nav_title: 'Work with an oracle',
-      heading: 'Advanced Challenge 3',
+      heading: 'Advanced Challenge: Conditional Hash Locked Transaction',
```
Comment (benalleng) [inline i18n/locales/en.ts:2601]:
```suggestion
      heading: 'Secret preimage locked transaction',
```

**Review by benalleng — COMMENTED** [review]
Looks great just going to remove the `Advanced Challenge` text as i think that is confusing with the possibility of difficulty modes.

**Comment by benalleng** [comment]
> This would simplify the styling, from styling based on some breakpoints to consistent padding across all screens.
> With this disclaimer, reviewing it should be less of a chore, and I can proceed with that.
> Is this fine?

Lets do this on a new PR otherwise great idea!

**Review by benalleng — APPROVED** [review]

**Pull request merged**